addons/weave/2.6.4/host-preflight.yaml

```diff
@@ -34,20 +34,14 @@ spec:
           - fail:
               when: "connection-refused"
               message: Connection to port 6781 was refused. This is likely to be a routing problem since this preflight configures a test server to listen on this port.
-          - warn:
+          - fail:
               when: "address-in-use"
               message: Another process was already listening on port 6781.
-          - fail:
-              when: "connection-timeout"
-              message: Timed out connecting to port 6781. Check your firewall.
           - fail:
               when: "error"
               message: Unexpected port status
           - pass:
```

Comment:
This no longer short-circuits. Everything needs to have a when.

Reply:
I just built and tested it with the latest preflight. It appears to be working as intended without a when.